charts/mailhog/templates/NOTES.txt

```diff
@@ -7,10 +7,10 @@ Web UI:
 =======
 
 export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "app={{ template "mailhog.name" . }},release={{ .Release.Name }}" -o jsonpath="{.items[0].metadata.name}")
-kubectl port-forward --namespace {{ .Release.Namespace }} $POD_NAME 8025
+kubectl port-forward --namespace {{ .Release.Namespace }} $POD_NAME {{ .Values.service.port.http }}
```

Comment:
That's not correct. The ports used here are container ports which are hard-coded, not service port.